src/test/java/co/omise/ClientTest.java

```diff
@@ -21,7 +21,7 @@
 
     @Test(expected = NullPointerException.class)
     public void testCreator() throws ClientException {
-        new Client(null);
+        new Client((String) null);
```

Comment:
Catch `null` look a bit wried to me. Do we have to do this?

Reply:
Hmmm I can change it to "", works better?

Reply:
Can we just delete `(String)`?

Reply:
No, it errors out

Reply:
😂Alright, let's keep it this way for now